source/renderer/app/stores/HardwareWalletsStore.ts

```diff
@@ -1652,9 +1652,9 @@ export default class HardwareWalletsStore extends Store {
         // Keep isTransactionInitiated active & Set new device listener by initiating transaction
         // Show message to reconnect proper software wallet device pair
         runInAction(
-          'HardwareWalletsStore:: set HW device CONNECTING FAILED',
+          'HardwareWalletsStore:: Unrecognized wallet (wrong passphrase)',
```

Comment:
@marcin-mazurek you should check the error code here. Connect can fail because of different reasons and mostly because there is no device on the provided path ( when local storage is corrupted )